lib/logmsg/logmsg-serialize.c

```diff
@@ -36,19 +36,40 @@
 
 #define OLD_LMM_REF_MATCH 0x0001
 
+static void
+_set_ts_processed(LogStamp *timestamps, const LogStamp *processed)
```

Comment:
Optional: the naming is a little unfortunate, as this is not a set function. `_init_ts_processed` or `_setup_ts_processed` perhaps?